### PR TITLE
[WIP] Modal keyboard cheatsheet

### PR DESCRIPTION
**Pull request by snidelyhazel**
+ Created scripts/modal.js, styles/_modal.scss, views/partials/modal.html
+ Updated scripts/index.js, styles/index.scss, views/layouts/main.html to include modal files

+ Bonus: corrected minor typo in bower_components/basecoat.readme.md

### CODE REVIEW

**Comment by zeke** [comment]
Looks like the actual tests are passing but CI is failing because of linting issues. Try running `standard --fix` locally which will automatically fix some of the formatting issues. The rest you may have to fix manually.

**Comment by zeke** [comment]
@snidelyhazel this is looking great. I am traveling this week but will review it in detail soon.

File: bower_components/basecoat/README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Basecoat sites
 
-A set of SCSS partials that work with the existing [Primer](https://github.com/primer/primer) toolkit to create resuable code and design patterns for GitHub's marketing sites.
```
[the remark this answers is not in the document]
Comment (snidelyhazel) [reply bower_components/basecoat/README.md]: I was waiting on the review from @zeke, but if you can let me know what else needs done before it's ready to go, that's great too!

**Comment by HashimotoYT** [comment]
@snidelyhazel I hope this pr is still alive, if is it, can you please rebase or merge from master for start review process?

**Comment by BinaryMuse** [comment]
Thanks for the PR! This was finished up in #2088

**Pull request closed without merging**